modules/apps/redirect/redirect-service/src/test/java/com/liferay/redirect/internal/provider/RedirectProviderImplTest.java

```diff
@@ -55,6 +56,8 @@ public void setUp() {
 		).thenReturn(
 			null
 		);
+
+		PropsUtil.set("feature.flag.LPS-160332", Boolean.TRUE.toString());
```

Comment:
Can this create a side effect if not put to false after the tests?

Reply:
No, this is a unit test, not an integration test, so no side effects.